python/pyspark/sql/functions.py

```diff
@@ -1802,10 +1802,10 @@ def from_json(col, schema, options={}):
 @since(2.1)
 def to_json(col, options={}):
     """
-    Converts a column containing a [[StructType]] into a JSON string. Throws an exception,
-    in the case of an unsupported type.
+    Converts a column containing a [[StructType]] or [[ArrayType]] into a JSON string.
+    Throws an exception, in the case of an unsupported type.
```

Comment:
If you don't strongly feel about this, let me leave as is (and sweep it in another PR). In most cases, I believe users will use directly with `select`.